optee-utee/optee-utee-sys/src/utee_syscalls.rs

```diff
@@ -2,10 +2,10 @@ use super::*;
 use libc::*;
 
 extern "C" {
-    pub fn utee_return(ret: c_ulong);
-    pub fn utee_log(buf: *const c_void, len: size_t);
-    pub fn utee_panic(code: c_ulong);
-    pub fn utee_get_property(
+    pub fn _utee_return(ret: c_ulong);
```

Comment:
`_utee_return` is also a non-return function. Maybe we should change this to:
```
pub fn _utee_return(ret: c_ulong) -> !;
```

Reference: https://github.com/OP-TEE/optee_os/blob/master/lib/libutee/include/utee_syscalls.h#L32